bin/utils/moonkey/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "moonkey"
+description = 'Moonbeam Key Tool'
+version = "0.1.0"
+authors = ["PureStake"]
+edition = "2018"
+license = 'GPL-3.0-only'
+homepage = 'https://moonbeam.network'
+
+[[bin]]
+name = "moonkey"
+path = "src/main.rs"
+
+[dependencies]
+clap = "3.0.0-beta.2"
```

Comment:
We're using structopt in the main moonbeam binary and other Substrate projects. crates.io/crates/structopt